.travis.yml

```diff
@@ -10,6 +10,12 @@ matrix:
       script:
         - 
           go get github.com/alecthomas/gometalinter && gometalinter --install && gometalinter ./... --deadline=10m --exclude=client/internal/client_helper.go
+    - os: linux
+      env: 
+        - todos
+        - install: true # Skip installing go packages
```

Comment:
No, this needs to be at the same level as script.

```yaml
env:
  - todos
install: true # Skip installing go packages
script:
...
```

Reply:
Sorry -- fixed